### PR TITLE
Update CONTRIBUTORS.md

### PR DESCRIPTION
**Pull request by reneisrael**
Adding my name

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -1396,16 +1401,6 @@ Noida
 - Bio: Front-end developer
 - GitHub: [Lisa Nguyen](https://github.com/LisaNguyen)
 
-### Name: [Peter Squicciarini](https://github.com/stripedpajamas)
```
Comment (StevenCopeland) [inline CONTRIBUTORS.md]: Why are you deleting these names????

Comment (reneisrael) [reply CONTRIBUTORS.md]: Sorry, it´s my first time

Comment (StevenCopeland) [reply CONTRIBUTORS.md]: Do you think you could create a commit adding them back?

Comment (reneisrael) [reply CONTRIBUTORS.md]: Sure, let me figure it out

Comment (StevenCopeland) [reply CONTRIBUTORS.md]:
```
#### Name: [Türker Yıldırım](https://github.com/turkerdotpy)		
 - Place: Tekirdağ, Turkey		
 - Bio: Literally gamer, geek and viking.		
 - GitHub: [turkerdotpy](https://github.com/turkerdotpy)		
```
Here they are. Copy the code into the readme wherever you see fit(just not in between contributions).

**Comment by Roshanjossey** [comment]
Hi  @reneisrael, I'm working on a similar project to help beginners to get started with contributing to open source projects.

Do check it out https://github.com/Roshanjossey/first-contributions

**Pull request merged**